src/common/createSlider.jsx

```diff
@@ -171,11 +171,6 @@ export default function createSlider(Component) {
       /* eslint-enable no-unused-expressions */
     }
 
-    onMouseUp = () => {
-      this.onEnd();
-      this.removeDocumentEvents();
-    }
```

Comment:
Why is this callback removed? Will cause a memory leak if we don't remove listeners from the document.

Reply:
Nevermind, listeners will be removed in `onMouseDown`.

Reply:
No, not `onMouseDown`. 
Actually, listeners will be removed in `this.onEnd`,  when another `mouseup` event  that bind on Document was trigger.

So, `mouseUp` event bind on `Slide` is repeated to another `mouseUp` bind on document.(trace: #361 )

https://github.com/react-component/slider/blob/c7f6f228d4d8a681a0d5d236d243898dda8e6421/src/common/createSlider.jsx#L161
https://github.com/react-component/slider/blob/c7f6f228d4d8a681a0d5d236d243898dda8e6421/src/Range.jsx#L111-L115